resolwe_bio/management/commands/generate_samples.py

```diff
@@ -0,0 +1,258 @@
+import gzip
+import io
+import json
+import os
+import random
+import shutil
+import string
+import zipfile
+
+from django.core.management.base import BaseCommand
+from django.contrib.auth.models import User
+from django.conf import settings
+from resolwe.flow.models import Data, DescriptorSchema, Process, Collection, Storage
+from resolwe_bio.tools.utils import escape_mongokey, gzopen
+
+
+class Command(BaseCommand):
+
+    """Generate test data"""
+
+    help = 'Generate test data'
+
+    def add_arguments(self, parser):
+        parser.add_argument('-a', '--n-annotated', type=int,
+                            required=False, default=15, help="Number of annotated samples to generate")
+        parser.add_argument('-u', '--n-unannotated', type=int,
+                            required=False, default=5, help="Number of unannotated samples to generate")
+
+    def word(self, length):
```

Comment:
It think it would be more descriptive to use `get_random_word` for the method name.

Also, either define the method as [`@staticmethod`](http://stackoverflow.com/questions/12179271/python-classmethod-and-staticmethod-for-beginner) or put it outside the `class`.